src/capybara.js

```diff
@@ -269,8 +269,20 @@ Capybara = {
     return true;
   },
 
+  sendKeys: function (index, keys) {
+    var strindex, length;
+
+    length = keys.length;
+
+    this.focus(index);
```

Comment:
This will end up sending two focus events if the keys are empty in `set`. Perhaps just wrap the focus and `node.value = ""` in an if block in `set`, and then use this shared implementation?